tests/framework/clients/rancher/client.go

```diff
@@ -42,6 +42,8 @@ func NewClient(bearerToken string, session *session.Session) (*Client, error) {
 		RancherConfig: rancherConfig,
 	}
 
+	session.HasCleanupSet = *rancherConfig.Cleanup
```

Comment:
```suggestion
	session.Cleanup = *rancherConfig.Cleanup
```

Reply:
With the bool name as Cleanup its name collides with our session.Cleanup func name. I think it should be named differently.

Reply:
fixed